app/js/components/search.jsx

```diff
@@ -91,14 +91,15 @@ export default class Search extends React.Component {
           <div className="col-sm-12 section search">
             {this.state.patients.length > 0 ?
               <div>
-                <h3>Search result </h3>  
-                <span className="spanable">Sort table by clicking on the header.
-                  Resize by dragging column.
-                  Click on a patient to view.</span> 
-                  <p/>  
+                <h3>Search result </h3>
```

Comment:
I feel that this text at the top just wastes space which could have been used to display search results. What do you think?

Reply:
I think you are right, especially since this is a data management/administrative tool the narration may be un necessary because the user may not require hand-holding..and yes, the space would be better used for the actual results.
Let me go ahead and claim back the space.

Reply:
@dkayiwa I have removed the text.